Doctrine/Phpcr/Page.php

```diff
@@ -45,13 +45,13 @@ class Page extends Route implements
     TranslatableInterface
 {
     /**
-     * @var NodeInterface
+     * @var PHPCRNodeInterface
```

Comment:
can you please rebase on master? i did a separate PR with this fix that i just merged.